collect_app/src/main/java/org/odk/collect/android/projects/ProjectSettingsDialog.kt

```diff
@@ -82,7 +82,7 @@ class ProjectSettingsDialog : DialogFragment() {
             dismiss()
         }
 
-        return AlertDialog.Builder(requireContext())
+        return MaterialAlertDialogBuilder(requireContext())
```

Comment:
We should really be using this everywhere, but didn't want to go down that route here.